R/Rules-class.R

```diff
@@ -448,6 +448,77 @@ NextBestInfTheory <- function(target, asymmetry) {
   .NextBestInfTheory(target = target, asymmetry = asymmetry)
 }
 
+# NextBestTDsamples ----
+
+## class ----
+
+#' `NextBestTDsamples`
+#'
+#' @description `r lifecycle::badge("stable")`
+#'
+#' [`NextBestTDsamples`] is the class to find a next best dose based on Pseudo
+#' DLT model with samples. Namely, it is to find two next best doses, one
+#' for allocation during the trial and the second for final recommendation at
+#' the end of a trial. Hence, there are two target probabilities of the
+#' occurrence of a DLT that must be specified: target probability to be used
+#' during the trial and target probability to be used at the end of the trial.
+#'
+#' @slot prob_target_drt (`proportion`)\cr the target probability of the
+#'   occurrence of a DLT to be used during the trial.
+#' @slot prob_target_eot (`proportion`)\cr the target probability of the
+#'   occurrence of a DLT to be used at the end of the trial.
+#' @slot derive (`function`)\cr the function which derives, based on a vector of
```

Comment:
```suggestion
#' @slot derive (`function`)\cr derives, based on a vector of
```

Reply:
The `nextBest` method for `NextBestTDsamples` is already cleaned up and tested. 
I apologise, I am not sure if I understand this particular change request. The `derive` slot is a function. Should I change the description somehow?